apps/richie/templates/app/dc.yml.j2

```diff
@@ -9,7 +9,7 @@ metadata:
   name: "richie-app-{{ deployment_stamp }}"
   namespace: "{{ project_name }}"
 spec:
-  replicas: {{ richie_app_replicas }}
+  replicas: { { richie_app_replicas } }
```

Comment:
Wow. Does-it work or did the linter make fun of you ?

Reply:
Oops. Linter's fault.